packages/datetime/src/common/datePickerBaseProps.ts

```diff
@@ -70,11 +70,6 @@ export interface DatePickerBaseProps {
      */
     locale?: string;
 
-    /**
-     * Collection of functions that provide internationalization support.
-     */
-    localeUtils?: typeof LocaleUtils;
```

Comment:
`LocaleUtils` is no longer available from `react-day-picker`, and thus `localeUtils` is no longer accepted as a prop.

See: https://github.com/palantir/blueprint/wiki/react-day-picker-8-migration#locale